src/mbgl/text/placement.cpp

```diff
@@ -289,7 +289,7 @@ float Placement::symbolFadeChange(TimePoint now) const {
 }
 
 bool Placement::hasTransitions(TimePoint now) const {
-    return symbolFadeChange(now) < 1.0;
+    return symbolFadeChange(now) < 1.0 || stale;
```

Comment:
In `MapMode::Still`, I think this should always be false (previously accomplished by the check in `symbolFadeChange`.

Reply:
`stale` should never be true in still mode now that placement is done on every frame